doc/sphinx/source/conf.py

```diff
@@ -256,7 +256,7 @@
 # One entry per manual page. List of tuples
 # (source start file, name, description, authors, manual section).
 man_pages = [
-    ('index', 'chpldoc', u'chpldoc Documentation',
+    ('index', 'chapel', u'Chapel Documentation',
```

Comment:
This looks like you're replacing the chpldoc man page, but I thought we already built one for chpldoc and one for chpl?  How is the chpldoc man page supported?